### PR TITLE
Genesis doc and test setup for bns testnet

### LINKED ISSUE #377: Document config file for zebranet
The config file for zebranet should be made to model the general setup we see for mainnet. We should produce a configuration file structure that we document, even if we change the actual public keys referred to.

Along with the genesis file should be a short document explaining it, so we can get feedback to business as to which elements they can easily tune in their discussions and what require code changes.

eg:
* creating the governance multisigs
* creating the validator distribution
* creating the block reward contract
* wiring them all together
* setting product fees
* gconf settings they can tweak (eg. anti-spam fee)
* and anything else similar that makes sense

**Acceptance Criteria**

- [x] `genesis.json` (with placeholder addresses, AAA... BBB... etc) created with realistic values for mvp usecase (in `cmd/bnsd/client/testdata/genesis.json`)
- [x] embed comments in `genesis.json` explaining the purpose and what can be tweaked.
- [ ] Use same format for the genesis fixtures (`cmd/bnsd/app/testdata/fixtures/app.go`)
- [x] Update AT to cover a simple usecase... product fee paid, block reward released, fees distributed to validators

### PR DESCRIPTION
**Pull request by alpe**
Resolves #377 

Test genesis and doc for the zebranet changes.

### CODE REVIEW

**Comment by ethanfrey** [comment]
What is left to do here? Is it ready for review?

**Comment by husio** [comment]
@alpe I am just merging support for a human-readable coin format https://github.com/iov-one/weave/pull/436
This will simplify the coin declaration as you cant write `{"fee": "1.2 IOV"}` and the below explanation is no longer needed
 > * `"fractional": 100000000`= 1/10 of a whole

File: cmd/bnsd/client/testdata/genesis_json.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  "cash": [
+    {
+      "address": "E28AE9A6EB94FC88B73EB7CBD6B87BF93EB9BEF0",
+      "coins": [
+        {
+          "ticker": "123456789 IOV"
+        },
+        {
+          "ticker": "100.1 ALX"
+        }
```
Comment (husio) [inline cmd/bnsd/client/testdata/genesis_json.md]: This should be 
```json
"coins": [
    "123456789 IOV", 
    "100.1 ALX"
]
```

File: cmd/bnsd/app/testdata/fixtures/app.go [hunk 2]
```diff
@@ -65,14 +64,7 @@ func appStateGenesis(keyAddress weave.Address) []byte {
...
 				"coins": []interface{}{
-					dict{
-						"whole":  50000,
-						"ticker": "ETH",
-					},
-					dict{
-						"whole":  1234,
-						"ticker": "FRNK",
-					},
+					"50000 ETH", "1234 FRNK",
```
Comment (ethanfrey) [inline cmd/bnsd/app/testdata/fixtures/app.go]: Nice :)

File: cmd/bnsd/client/testdata/genesis_json.md [hunk 3]
```diff
@@ -20,13 +20,10 @@ A is an address for a public key or contract with currency tokens filled.
       "address": "E28AE9A6EB94FC88B73EB7CBD6B87BF93EB9BEF0",
       "coins": [
         {
-          "ticker": "IOV",
-          "whole": 123456789
+          "ticker": "123456789 IOV"
```
Comment (ethanfrey) [inline cmd/bnsd/client/testdata/genesis_json.md]: This was not tested, was it?

Comment (alpe) [reply cmd/bnsd/client/testdata/genesis_json.md]: Unfortunately I can not test this with the genesis.json file in this directory as the client uses a different model for deserialization. I had reverted the changes there but these 2 lines are just wrong. Good finding!

**Pull request merged**